src/Blocks/components/table/table.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Template for the Table Component.
+ *
+ * @package EightshiftForms
+ */
+
+use EightshiftFormsVendor\EightshiftLibs\Helpers\Helpers;
+
+$manifest = Helpers::getManifestByDir(__DIR__);
+
+$componentClass = $manifest['componentClass'] ?? '';
+$additionalClass = $attributes['additionalClass'] ?? '';
+$selectorClass = $attributes['selectorClass'] ?? $componentClass;
+
+$tableContent = Helpers::checkAttr('tableContent', $attributes, $manifest);
+$tableHead = Helpers::checkAttr('tableHead', $attributes, $manifest);
+
+$tableWrapClass = Helpers::classnames([
+	Helpers::selector($componentClass, $componentClass),
+	Helpers::selector($selectorClass, $selectorClass, $componentClass),
+	Helpers::selector($additionalClass, $additionalClass),
+]);
+
+$tableClass = Helpers::classnames([
+	Helpers::selector($componentClass, $componentClass, 'table'),
+]);
+
+?>
+
+<div class="<?php echo esc_attr($tableWrapClass); ?>">
+	<table>
+		<?php if ($tableHead) { ?>
+			<thead>
+				<tr>
+					<?php foreach ($tableHead as $head) { ?>
+						<th><?php echo $head; // phpcs:ignore Eightshift.Security.HelpersEscape.OutputNotEscaped ?></th>
+					<?php } ?>
+				</tr>
+			</thead>
+		<?php } ?>
+		<?php if ($tableContent) { ?>
```

Comment:
Should anything even be rendered if the tableContent is empty?

Reply:
parent logic is responsible for this so it is ok